cmd/frakti/frakti.go

```diff
@@ -59,3 +62,14 @@ func main() {
 
 	fmt.Println(server.Serve(*listen))
 }
+
+func getStreamingConfig() *streaming.Config {
+	config := &streaming.Config{
+		Addr:                  "localhost:" + *streamingServerPort,
```

Comment:
This shouldn't be localhost, kube-apiserver needs this endpoint for communication.